src/components/WindowChildren/WindowTerminal.vue

```diff
@@ -298,6 +321,16 @@ export default {
       this.input_text += val + '\n'
       this.protection_length += val.length + 1
     },
+    cmd_clear_last_line(){//replace last line with header
+      const lastItr = this.input_text.lastIndexOf('\n');
+      if(lastItr==-1){
+        this.input_text = this.header
+      }else{
+        this.input_text = this.input_text.slice(0,lastItr+1)+this.header
+        this.protection_length = this.header.length
+      }
+        this.protection_length = this.input_text.length
+    },
```

Comment:
I think it's OK to use just
```javascript
  cmd_clear_last_line(){
      //replace last line with header
      this.input_text = this.input_text.substr(0,this.protection_length)
    },
```
thus makes it no point to be separate as a method